tapdb/multiverse.go

```diff
@@ -152,6 +152,15 @@ func (b *MultiverseStore) RootNodes(
 				groupedAssets map[asset.ID]uint64
 			)
 
+			// Parse universe proof type and populate the universe
+			// ID.
+			id.ProofType, err = universe.ParseStrProofType(
```

Comment:
Makes sense, lead to blank values for grouped assets for the map we added. 

Also needs a test.